src/models/IOkResponse.ts

```diff
@@ -0,0 +1,4 @@
+export interface IOkResponse {
+  status?: number;
+  data?: 'OK'
+}
```

Comment:
❗️
Можно обойтись одним интерфейсом на респонс. И почему везде `?`?
```suggestion
export interface IResponse {
  status?: number;
  data?: 'OK' | { reason?: string; id?: number }
}
```

Reply:
Исправлено!